### PR TITLE
added saved preferences animation

### PR DESCRIPTION
**Pull request by Kirazuto7**
Files Changed:

App.js: Changed the landing page based on whether the user is logged into firebase or not. If logged in then they land on the card page otherwise they land on the login page. (User may persist log in state until they sign out or switch to a different browser/device)

Preferences: Added a popup view that appears when the user updates their preferences to notify the user that they saved their changes successfully. The view will disappear after 2 seconds.

### CODE REVIEW

**Comment by beaverseven** [comment]
Good job changing the routing to initially set it to the login screen when authenticated. I like how you utilized Preferences.js to dynamically update preferences when changes are saved. Specifically the div that it returns with the css fits perfectly. Good job!

File: my-app/src/Components/App.js
Comment (chizy7) [inline my-app/src/Components/App.js:1]: This looks good for the routing

**Review by chizy7 — APPROVED** [review]
Good job on this!

**Review by JChen0915 — APPROVED** [review]
Use of states for providing the animation seems like a good implementation. Good job

**Review by beaverseven — APPROVED** [review]
The Routing changes look good for setting the users default route when they're logged in.

**Pull request merged**